README.md

```diff
@@ -14,9 +14,26 @@ A CLI to interact with the Svix API.
 
 ## Installation
 
-Homebrew Tap & Scoop Bucket Coming soon.
+### macOS
 
-For now, you can download the binary executable and use it straight away without having to install any additional dependencies.
+The Svix CLI is available on macOS via [Homebrew](https://brew.sh/):
+
+```sh
+brew install svixhq/svix/svix
```

Comment:
Is it missing the `tap add` or is it good like this?

Reply:
It's good like this, this is just a oneliner,`brew install <tap>/<fomula>`
the alternative would be
```sh
brew tap svixhq/svix
brew install svix
```

Reply:
Cool. :)